src/com/intimetec/constants/Constants.java

```diff
@@ -0,0 +1,43 @@
+package com.intimetec.constants;
```

Comment:
1. Proper Grammar  rules not followed ,  sentence should start with capital latter, 
2. Duplicate Constants